lib/shelley/exe/cardano-wallet.hs

```diff
@@ -223,6 +225,11 @@ cmdServe = command "serve" $ info (helper <*> helper' <*> cmd) $ mempty
                                 exitWith $ ExitFailure 33
 
                 whenJust databaseDir $ setupDirectory (logInfo tr . MsgSetupDatabases)
+                let socket = T.pack nodeSocket
+                when (os == "mingw32" && not ("\\\\.\\pipe\\" `T.isPrefixOf` socket)) $ do
+                    logError tr (MsgInvalidWinNamedPipe socket)
+                    exitWith $ ExitFailure 34
```

Comment:
This validation is now redundant with the validation done in the command-line I believe :thinking: